### ISSUE TITLE
Operation is not supported on this platform exception in .Net Standard

### ISSUE THREAD

**Issue by nologinatgit** [thepirat000/Audit.NET#180]
**Describe the bug**
I implemented the audit in a _.Net Standard 2.0_ project and used it in my _Xamarin.Android_ project. 

```
Configuration.Setup()
.UseEntityFramework(_ => _.AuditTypeMapper(t=> typeof(AuditLog))
.AuditEntityAction<AuditLog>((ev, entity, entry) =>
{
   entity.ID = Guid.NewGuid();
   entity.Time= DateTime.UtcNow;           
   entity.Table = entry.Table;
   entity.EType = entry.Action;
   entity.User = "Test User";
}).IgnoreMatchedProperties(true));
...
auditDbContext = new DefaultAuditContext(this)
{
   Mode = AuditOptionMode.OptIn 
};
dbContextHelper.SetConfig(auditDbContext);

```

Saving to SQLite database was no problem with Entity Framework Core (it supports .Net Standard 2.0), but when I configured Audit.EntityFramework, it suddenly started to throw this exception, _even with entities that are not marked for audit_:

```
System.PlatformNotSupportedException: Operation is not supported on this platform.
  at System.Runtime.InteropServices.Marshal.GetExceptionCode () [0x00000] in <43dbbdc147f2482093d8409abb04c233>:0 
  at Audit.Core.AuditScope.GetCurrentException () [0x00001] in <a8301062ea114e2397cd5b70a581ae20>:0 
  at Audit.Core.AuditScope.EndEvent () [0x00001] in <a8301062ea114e2397cd5b70a581ae20>:0 
  at Audit.Core.AuditScope+<SaveAsync>d__34.MoveNext () [0x00024] in <a8301062ea114e2397cd5b70a581ae20>
```

**To Reproduce**
Add Audit.EntityFramework to a .Net Standard project and try to use it.

**Expected behavior**
Everything is saved and works normally.

**Libraries (specify the Audit.NET extensions being used including version):**
For example:
 - Audit.EntityFramework: 13.2.2
 - Audit.EntityFramework.Core: 13.2.2 (also tried)

**Additional context**
Should I use some other configuration for .Net Standard and Xamarin?

**Comment by thepirat000** [comment]
.NET standard is not a problem, the problem is with Xamarin (Mono). 

The library tries to obtain the current exception with a method that is not supported on Mono. Check [this line](https://github.com/mono/mono/blob/master/mcs/class/corlib/System.Runtime.InteropServices/Marshal.cs#L521).

I will take a deeper look to avoid the problem

**Comment by nologinatgit** [comment]
Thank you.
If you need more information, do tell please. I can't think of anything that should affect the previous saving mechanism.

**Comment by thepirat000** [comment]
This was fixed on version 13.3.0, please upgrade your references and try again. Comment if you're still having problems,

**Comment by nologinatgit** [comment]
Thanks for the quick fix, it works as expected :)
